helix-term/src/commands/typed.rs

```diff
@@ -2199,6 +2199,47 @@ fn clear_register(
     Ok(())
 }
 
+pub fn process_cmd(
```

Comment:
This doesn't need to be a function on its own, it can be moved back into the prompt definition in `command_mode`